### PR TITLE
Workspace nikhil

### PR DESCRIPTION
**Pull request by nikhilfusion**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -87,3 +87,5 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+src/nv_midbox_websock/templates/static/bower_components/
```
Comment (sugchand) [inline .gitignore:91]: Do not add anything in the gitignore.
Its not a good practice.
Either keep those files in the source tree or provide option to install it.

File: src/nv_midbox_websock/templates/.bowerrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "directory": "static/bower_components"
```
Comment (sugchand) [inline src/nv_midbox_websock/templates/.bowerrc:2]: . file?

Do you really think we need to use the bower for packaging? if there is only few file. I would suggest to get rid of the package manager and keep the file in the source tree statically.

File: src/nv_midbox_websock/templates/bower.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "NeoviewApp",
+  "homepage": "https://github.com/sugchand/neoview-middle-box",
+  "authors": [
+    "Nikhil T Nair"
+  ],
+  "description": "",
+  "main": "",
+  "license": "MIT",
```
Comment (sugchand) [inline src/nv_midbox_websock/templates/bower.json:9]: why MIT

File: src/nv_midbox_websock/templates/static/js/app.js [hunk 4]
```diff
@@ -1,47 +1,43 @@
...
-      description: "Bedside camera"
-    },{
-      id : 6,
-      name: "Camera6",
-      status: 1,
-      description: "Test one"
-    }
-  ];
-  console.log("this is the controller");
+  // $scope.cameraInfo = [
```
Comment (sugchand) [inline src/nv_midbox_websock/templates/static/js/app.js:10]: Remove all commented out fields if its not necessary

**Review by sugchand — CHANGES_REQUESTED** [review]
Change the files as suggested..

**Comment by nikhilfusion** [comment]
close

**Pull request closed without merging**